### ISSUE TITLE
Notification via Telegram

### ISSUE THREAD

**Issue by Ansem93** [pyouroboros/ouroboros#146]
Hi everyone!
Do you think that it would be possible to add Telegram notification? I'm recieving all my server notification on my bot right now, and I love it. I think it's a lot better than classic email.

**Comment by DirtyCajunRice** [comment]
telegram is a webhook with different post formatting, so absolutely! we will need to put some defining feature to ensure it only formats the data the telegram way when sent to a telegram endpoint. like with healthchecks i think it would be safest to assume “telegram” needs to be in the url.

does telegram have any fancy formatting ability or is it like pushover in that its plaintext

**Comment by Ansem93** [comment]
It should be plaintext for what I know.
On OMV they just send the same text via email and via Telegram.
Here's an example:
```
CRON-APT completed on Delibird [/etc/cron-apt/config] on the Fri, 25 Jan 2019 00:20:10 +0100 (CET) 

 CRON-APT RUN [/etc/cron-apt/config]: Fri Jan 25 00:07:16 CET 2019
CRON-APT SLEEP: 752, Fri Jan 25 00:19:48 CET 2019
CRON-APT ACTION: 3-download
CRON-APT LINE: /usr/bin/apt-get -o Acquire::http::Dl-Limit=25 dist-upgrade -d -y -o APT::Get::Show-Upgraded=true
Reading package lists...
Building dependency tree...
Reading state information...
Calculating upgrade...
The following NEW packages will be installed:
  linux-image-4.19.0-0.bpo.1-amd64
The following packages will be upgraded:
  apt apt-transport-https apt-utils base-files libapt-inst2.0 libapt-pkg5.0
  libsystemd0 libudev1 linux-image-amd64 openmediavault-omvextrasorg systemd
  systemd-sysv udev
13 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 75.2 kB/53.8 MB of archives.
After this operation, 261 MB of additional disk space will be used.
Get:1 https://dl.bintray.com/openmediavault-plugin-developers/arrakis stretch/main amd64 openmediavault-omvextrasorg all 4.1.14 [75.2 kB]
Fetched 75.2 kB in 3s (23.6 kB/s)
Download complete and in download only mode
```

**Comment by DirtyCajunRice** [comment]
@sentriz I really like that project and had never heard of it before. We are looking into it

**Comment by sentriz** [comment]
@DirtyCajunRice it is very good, I use it for one of my projects. Also the developer @caronc is very helpful

**Comment by caronc** [comment]
Thank you for the praise @sentriz .  @DirtyCajunRice by all means please let me know if i can help you out in any way.

**Comment by DirtyCajunRice** [comment]
@caronc Theres a few confusing things. The basics all look normal but id like to not lose the functionality of a few things we get with some platforms like discord. If you have some time id love if you could jump into our discord and ping me

**Comment by caronc** [comment]
Sorry, I'm just seeing your comment now.  I realize we've got 2 dialogues going on (one [here](https://github.com/caronc/apprise/issues/10) too).

You're right with your assumption though; _apprise_ its current form would definitely starve you of advance features.  That was the sacrifice made to support as many services as possible unfortunately.  I've tried to accommodate some specific cases through the notification URL itself.  So it's not to say we can't try and accommodate ideas you have; I guess it depends on what they are. :slightly_smiling_face:
